developer-community/fix-DCO.md

```diff
@@ -109,6 +109,18 @@ grep -qs "^${SIGNATURE}" "$1" || echo "\n${SIGNATURE}" >> "$1"
 
 You need to do this for each repository independently.
 
+If you want to create this hook **automatically** on every new git repositories you will create from now on (using `git init`, `git clone`...) you can use `git-templates`; to do so, just save the script described above into (duplicate files):
+- `~/.git-templates/hooks/prepare-commit-msg`
+- `~/.git-templates/hooks/commit-msg`
+
+And configure it globally:
+
+```bash
+git config --global init.templatedir ~/.git-templates
+```
+
+**note:** you will also need to paste this hook in your existent git repositories.
```

Comment:
**NOTE**: You will still need to paste this hook in your existing git repositories.